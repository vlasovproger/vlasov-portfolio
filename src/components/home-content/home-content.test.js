import HomeContent from "./home-content";
import { render } from "@testing-library/react";

describe("HomeContent", () => {
  it("should render component", () => {
    const container = render(<HomeContent />);
    expect(container).toMatchSnapshot();
  });
});
