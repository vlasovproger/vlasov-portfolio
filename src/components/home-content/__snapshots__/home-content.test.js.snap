// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContent should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content"
      >
        <div
          class="vertical-lines"
        >
          <div
            class="vertical-line"
          />
          <div
            class="vertical-line colored"
          />
          <div
            class="vertical-line colored"
          />
          <div
            class="vertical-line colored"
          />
          <div
            class="vertical-line"
          />
        </div>
        <h1>
          Andrei Vlasov.
        </h1>
        <div
          class="sub"
        >
          <hr />
          <h2>
            Web Developer Portfolio.
          </h2>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content"
    >
      <div
        class="vertical-lines"
      >
        <div
          class="vertical-line"
        />
        <div
          class="vertical-line colored"
        />
        <div
          class="vertical-line colored"
        />
        <div
          class="vertical-line colored"
        />
        <div
          class="vertical-line"
        />
      </div>
      <h1>
        Andrei Vlasov.
      </h1>
      <div
        class="sub"
      >
        <hr />
        <h2>
          Web Developer Portfolio.
        </h2>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
